backend/app/core/search.py

```diff
@@ -570,7 +598,9 @@ def build_opensearch_request_body(
         if search_request.exact_match:
             builder.with_exact_query(search_request.query_string)
         else:
-            builder.with_semantic_query(search_request.query_string)
+            # Disable KNN search for sensitive query terms
+            use_knn = search_request.query_string.lower() not in sensitive_query_terms
```

Comment:
I went for `term == query` rather than `term in query` as longer terms have more meaning that we'd want to get picked up by vector search, and mean examples that don't look that great are less obvious. E.g., while the first result here isn't relevant (and would potentially be problematic for the query `transgender`), it's not a showstopper IMO, and the other results are relevant to the query

![image](https://user-images.githubusercontent.com/20212179/195131366-526c8527-29f5-492b-b482-d660cfd6017c.png)